eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -80,6 +82,30 @@ private void initClient(
       @Nullable ConfigurationStore configurationStoreOverride,
       String apiKey,
       boolean offlineMode) {
+    initClient(
```

Comment:
How come we need this extra method in this test. Seems like we could just pass null (or a mock) as desired

Reply:
there were sooo many callsites and I was being lazy.